sway/commands/resize.c

```diff
@@ -378,8 +378,8 @@ static struct cmd_results *resize_set_floating(struct sway_container *con,
 			con->pending.width = width->amount;
 			break;
 		case MOVEMENT_UNIT_INVALID:
-			sway_assert(false, "invalid width unit");
-			break;
```

Comment:
I'm not sure how this was reachable before, because the only caller is `cmd_resize_set()` and that function already checks for invalid width/height amounts?